doc/JSON_INFO.md

```diff
@@ -651,6 +653,11 @@ For information about tools with option to export ASCII art in format ready to b
 | learned_proficiencies       | (_optional_) Array of proficiency ids you gain when installing this CBM, and lose when unintalling
 | installation_requirement    | (_optional_) Requirment id pointing to a requirment defining the tools and componentsnt necessary to install this CBM.
 | vitamin_absorb_mod          | (_optional_) Modifier to vitamin absorbtion, affects all vitamins. (default: `1.0`)
+| cant_uninstal_msg           | (_optional_) The text shown when trying to uninstall a bionic which cannot be uninstalled. Should be with a possessive pronouns & a nominative one, both replaced with '%s'.
```

Comment:
Maybe they should be switched to `%1$s` and `%2$s` for tweaking where they appear?
```suggestion
| cant_uninstall_msg           | (_optional_) The text shown when trying to uninstall a bionic which cannot be uninstalled. The first %s in the string will be replaced with a possessive pronouns & the second a nominative one.
```